rtgui/colorappearance.h

```diff
@@ -29,12 +29,12 @@
 #include "colorprovider.h"
 
 class ColorAppearance final :
-    public ToolParamBlock,
-    public AdjusterListener,
-    public FoldableToolPanel,
-    public rtengine::AutoCamListener,
-    public CurveListener,
-    public ColorProvider
+        public ToolParamBlock,
```

Comment:
@Floessie what should I fix?
https://github.com/Beep6581/RawTherapee/blob/dev/rtgui/colorappearance.h#L32